moj-be/modules/custom/moj_healthcheck/src/Controller/HealthcheckApiController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\moj_healthcheck\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Database;
+
+class HealthcheckApiController extends ControllerBase
+{
+    public $response;
+
+    public function healthcheckApiEndpoint()
+    {
+        $this->response = $this->setResponse();
+        return $this->response;
+    }
+
+		public function setResponse()
+    {
+        return $this->databaseSimpleTest() ? time() : 'FALSE';
+    }
+
+		public function databaseSimpleTest()
+    {
+        $connection = \Drupal::database();
```

Comment:
what's the timeout set to for this? Ideally it should be <1 second.

Reply:
I've set this to 30 seconds